### DISCUSSION TITLE
What about the video?

### DISCUSSION THREAD

**Discussion by inson1** [z-huang/music#1116] [General]
I'm missing videos from YouTube music.
What are you thinking? Is it better without them?

**Comment by JPGuillemin** [comment]
Please no !!!!! Vidéo are useless for music listening and just a waste of battery and data.

And there already are many good yt video apps. No need for another bloat.

**Reply by inson1** [reply]
@JPGuillemin It would be only add on option.

**Comment by netrunner-exe** [comment]
> Please no !!!!! Vidéo are useless for music listening and just a waste of battery and data.
> 
> And there already are many good yt video apps. No need for another bloat.

In YM Revanced there is an option for clips to play the video instead of the album art (if playing a music video). This is a cool feature, if you don’t like it, it doesn’t mean that others don’t need it. I don't see anything wrong with it, I like it and use it a lot in YM Revanced. I guess that's what @inson1 meant, rather than playing YouTube videos in general.

**Reply by inson1** [reply]
@netrunner-exe yes, but I dont see the difference between clips and videos. I like amv for example. Is that clip? Video?

But yea, it should be add on.
